posthog/batch_exports/service.py

```diff
@@ -102,6 +103,7 @@ class BaseBatchExportInputs:
     backfill_details: BackfillDetails | None = None
     batch_export_model: BatchExportModel | None = None
     batch_export_schema: BatchExportSchema | None = None
+    integration_id: int | None = None
```

Comment:
Based on the model `posthog.models.integration.Integration` this is a `TextField`. Why do we type this as an `int`?

Reply:
it's a bit confusing but there are 2 different integration IDs: `integration.id` and `integration.integration_id`. We're referencing the former, which is the primary key of the table.

Reply:
Thanks for clarifying